class ThePain(Exception):
    pass


class UserNotFoundError(ThePain):
    pass
